tf_files/aws/batch/root.tf

```diff
@@ -0,0 +1,203 @@
+terraform {
+  backend "s3" {
+    encrypt = "true"
+  }
+}
+
+provider "aws" {}
+
+resource "aws_vpc" "new_vpc" {
+  cidr_block = "10.1.0.0/16"
+}
+
+resource "aws_internet_gateway" "gw" {
+  vpc_id = "${aws_vpc.new_vpc.id}"
+
+  tags = {
+    Name = "main"
+  }
+}
+
+resource "aws_subnet" "new_subnet" {
+  vpc_id     = "${aws_vpc.new_vpc.id}"
```

Comment:
Awesome - thanks for setting up a new VPC.  Is it possible to keep the subnets private, and add VPC endpoints for S3 and ECS?

Reply:
The instances need to download quay image from internet so we need public subnets